packages/sitecore-jss/src/personalization/personalization-decisions-service.ts

```diff
@@ -0,0 +1,175 @@
+import { AxiosDataFetcher } from './../axios-fetcher';
+import { fetchData, HttpDataFetcher } from './../data-fetcher';
+import debug from '../debug';
+import { extractQueryStringParams } from '../utils/extract-querystring-params';
+
+export interface RenderingPersonalizationDecision {
+  /**
+   * The variant key
+   */
+  variantKey?: string;
+  /**
+   * The error message
+   */
+  errorMessage?: string;
```

Comment:
Is this for devs to customize the error message? What is the use case for this?

Reply:
These errorMessages is not for end user, they are more for developers to identify invalid server response (e.g. pass invalid params to decision endpoint)
We will add console.debug to proper place.